.circleci/integration-tests/Dockerfile.astro_cloud

```
@@ -87,5 +87,6 @@ RUN cp -r example_* ${AIRFLOW_HOME}/dags
 RUN cp master_dag.py  ${AIRFLOW_HOME}/dags/
 RUN cp astronomer_migration_dag.py  ${AIRFLOW_HOME}/dags/
 RUN cp nuke-config.yml  ${AIRFLOW_HOME}/dags/
-
+# we have an issue with connexion==3.0.0, so for now pinning previous stable version
+RUN pip install connexion==2.14.2
```

```suggestion
RUN pip install connexion!=3.0.0
```

Like this we will also be test the new release out for connexion.

---

okay but not sure if 3.0.0 is already installed then whether my suggestion would reinstall other version or not :) so reverting my suggestion.